huggle/projectconfiguration.cpp

```diff
@@ -174,6 +174,7 @@ bool ProjectConfiguration::Parse(QString config, QString *reason, WikiSite *site
     // Welcoming
     this->Welcome = HuggleParser::ConfigurationParse("welcome", config);
     this->WelcomeMP = HuggleParser::ConfigurationParse("startup-message-location", config, "Project:Huggle/Message");
+    this->PageEmptyQueue = HuggleParser::ConfigurationParse("emptyqueue-message-location", config, "");
```

Comment:
Project configuration? Why? This should go to user configuration, we don't want to override this setting per whole project, it should be optional for users who want to change that cat.

Also if you really want people to use this feature, information that this page can be customized should be inserted to default page